test/e2e/rule_test.go

```diff
@@ -241,6 +241,16 @@ func (a *failingStoreAPI) Info(context.Context, *storepb.InfoRequest) (*storepb.
 				Value: "store_api",
 			},
 		},
+		LabelSets: []storepb.LabelSet{
+			{
+				Labels: []storepb.Label{
```

Comment:
Let's make test more ambitious and have multiple set here?

Reply:
Done